doctr/utils/visualization.py

```diff
@@ -229,3 +229,24 @@ def synthetize_page(
                 response[ymin:ymax, xmin:xmax, :] = np.array(img)
 
     return response
+
+
+def draw_boxes(boxes: np.ndarray, image: np.ndarray, **kwargs) -> None:
+    """Draw an array of relative straight boxes on an image
+
+    Args:
+        boxes: array of relative boxes, of shape (*, 4)
+        image: np array
+    """
+    h, w = image.shape[:2]
+    for box in boxes.tolist():
+        xmin, ymin, xmax, ymax = box
```

Comment:
the conversion to absolute coords could be vectorized by doing this for all boxes at once